src/m365/outlook/commands/roomlist/roomlist-list.ts

```diff
@@ -0,0 +1,45 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: GlobalOptions;
+}
+
+class OutlookRoomlistListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ROOMLIST_LIST;
+  }
+
+  public get description(): string {
+    return 'Get a collection of all available roomlist';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    return telemetryProps;
+  }
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'phone', 'emailAddress'];
+  }
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/places/microsoft.graph.roomlist`,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json'
+      },
+      responseType: 'json'
+    };
+
+    request.get(requestOptions)
```

Comment:
Once you are using the GraphItemsListCommand you can use the helper method getAllItems. This will help you traverse through all items

Reply:
The code is now using getAllItems method. Thank you!